specification/synapse/data-plane/readme.go.md

```diff
@@ -48,4 +50,26 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 go:  
     namespace: accesscontrol
 output-folder: $(go-sdk-folder)/services/preview/synapse/2020-02-01-preview/$(namespace)
+```
+
+### Tag: package-monitoring-2019-11-01-preview
+
+These settings apply only when `--tag=package-monitoring-2019-11-01-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-monitoring-2019-11-01-preview' && $(go)
+go:  
+    namespace: monitoring
+output-folder: $(go-sdk-folder)/services/preview/synapse/2019-11-01-preview/$(namespace)
+```
+
+### Tag:  package-vnet-2019-06-01-preview and go
+
+These settings apply only when `--tag=package-vnet-2019-06-01-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-vnet-2019-06-01-preview' && $(go)
+go:  
+    namespace: vnet
```

Comment:
change `vnet` to `managedVirtualNetwork` per PM's requirement

Reply:
Got it. I will update it.